### ISSUE TITLE
Environment Activator & Symfony 4.3+ "broken"

### ISSUE THREAD

**Issue by c33s** [bestit/flagception-bundle#65]
i updated my app from sf 3.4 to sf4.4 and my environment activators stopped working.  this is because of https://github.com/symfony/symfony/issues/31765

the `DotEnv` component get instanced differenlty in symfony 4.3 in the `bootstrap.php` file:

```
(new Dotenv(false))->loadEnv(dirname(__DIR__).'/.env');
for
(new Dotenv(true))->loadEnv(dirname(__DIR__).'/.env'); 
```
the `true` means that the variables of the `.env` files get "written" with `putenv`. this behavior was changed.

as flagception is using `getenv()` this change hit it.  

the ticket is just for reference or maybe for a refactor to load and inject the variabled with `%env()%` (if possible).

**Comment by migo315** [comment]
Thank for this issue and your great research!

I will create a fix for this in the next days. ;-)

**Comment by migo315** [comment]
> the ticket is just for reference or maybe for a refactor to load and inject the variabled with %env()% (if possible).

I think we should handle both. Fix the enviroment activator for symfony >=4.4 and also add the support for `%env()%`

**Comment by migo315** [comment]
I finished the changes and publish it under 3.4.0. Please remember that also the Flagception SDK must be updated:

`composer update flagception/flagception-bundle flagception/flagception`

After that, the `env` field should work again as expected. In addition, you can use the symfony `%env()%` syntax as alternative:

```yaml
# config.yml

flagception:
    features:      
        feature_123:
            default: '%env(FEATURE_NAME_FROM_ENV)%'
```

So I close the ticket. If you have questions or still encounter problems, you can just open it again.

Thanks again for your support. ;-)
